frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -407,11 +407,14 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     </>
   );
 
+  const singleNamespaceInstallModeFilter = (namespace) => namespace !== 'openshift-operators';
```

Comment:
We shouldn't hard-code the `openshift-operators` namespace. This isn't the right namespace for native Kubernetes. You could use `globalNS`:

https://github.com/spadgett/console/blob/c083a8cc189eb4fbb965ccca94c029ece71647d1/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx#L98-L101

I wonder if we should filter out any namespaces with an operator group that doesn't allow single namespace installation. But then it might not be clear why a namespace isn't in the list.

Should we just show a clear error message when this is picked? That might be the least confusing.

Reply:
@spadgett yeah, so other option would be to show an error msg, like 
`The namespace 'openshift-operators' does not support install modes for this operator.`
or whatever namespace that global.

Either removing them or showing the warning makes sense to me 👍

Reply:
I thought we already had the warning, though. Is it not showing up?

Reply:
<img width="586" alt="Screenshot 2020-04-07 at 18 25 36" src="https://user-images.githubusercontent.com/1668218/78694615-39342880-78fd-11ea-8187-a507813c0e5b.png">

only this one

Reply:
OK, so there's already an error. I think we just need to improve the message.